### PR TITLE
Bump latest Ruby versions

### PR DESCRIPTION
**Pull request by stanhu**
See https://www.ruby-lang.org/en/downloads/releases/

### CODE REVIEW

File: lib/parser/current.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def warn_syntax_deviation(feature, version)
     CurrentRuby = Ruby23
 
   when /^2\.4\./
-    current_version = '2.4.0'
+    current_version = '2.4.3'
```
Comment (whitequark) [inline lib/parser/current.rb]: Nope, we still don't even support 2.4.1. If you remove this change I'll merge the rest.

Comment (stanhu) [reply lib/parser/current.rb]: Interesting, thanks!

**Comment by stanhu** [comment]
Thanks! Would you mind releasing a new version?

**Comment by gitlab-winnie** [comment]
@whitequark Do you have a release planned including these changes?

**Comment by whitequark** [comment]
When I have time for it, yes.

**Pull request merged**